domain-rrd/rrd-app-test/spectral.yaml

```diff
@@ -0,0 +1,3 @@
+extends: spectral:oas
+rules:
+  operation-description: error
```

Comment:
When is this used? Do we still need the one in `app`?

Reply:
rm'd